cookbooks/creating_a_context_configuration_extension.rst

```diff
@@ -223,20 +223,28 @@ the ``HelloWorldExtension`` key and configure our ``text`` and ``enable`` value.
 
 Finally, we need to load the ``HelloWorld\Context\HelloWorldContext`` into our suite.
 
-Here's the ``behat.yaml``:
-
-.. code-block:: yaml
-
-  default:
-    suites:
-      default:
-        contexts:
-          - FeatureContext
-          - HelloWorld\Context\HelloWorldContext
-    extensions:
-      HelloWorld\ServiceContainer\HelloWorldExtension:
-        text: 'Hi there!'
-        enable: true
+Here's the ``behat.php``:
+
+.. code-block:: php
+
+    <?php
+
+    use Behat\Config\Config;
+    use Behat\Config\Extension;
+    use Behat\Config\Profile;
+    use Behat\Config\Suite;
+
+    return new Config()
+        ->withProfile(new Profile('default')
+            ->withExtension(new Extension('HelloWorld\ServiceContainer\HelloWorldExtension', [
```

Comment:
Maybe better to use FQCN::class now that we can (either inline, or with an import)?

```suggestion
            ->withExtension(new Extension(HelloWorld\ServiceContainer\HelloWorldExtension::class, [
```